README.md

```diff
@@ -1,56 +1,78 @@
-<a name="readme-top"></a>
-# Microsoft Teams Integration / Dataminer Bot
-
-This guideline is about how to integrate Dataminer into Microsoft Teams with automation scripts.
+<a id="readme-top"></a>
+
+# Chat Integration
+
+This guideline is about how to integrate Dataminer into business communication platforms with automation scripts.
+
+<details>
+<summary>Table of Contents</summary>
+  <ul>
+    <li><a href="#overview">Overview</a></li>
+    <li><a href="#prerequisites">Prerequisites</a></li>
+    <li><a href="#installation">Installation</a></li>
+    <li><a href="#examples">Examples</a></li>
+    <li><a href="#help">Help</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#version-history">Version History</a></li>
+    <li><a href="#license">License</a></li>
+  </ul>
+</details>
 
 ## Overview
 
-There are two main parts in this guideline:
- * Adding the automation scripts & memory files to your Dataminer system.
- * Granting admin consent in the DCP admin app.
+### Teams
+
+There are two main parts to the Chat Integration:
+
+- Adding the automation scripts & memory files to your Dataminer System.
+- Granting admin consent in the DCP admin app.
+
+## Prerequisites
+
+### General
 
-## Getting Started
+- The DataMiner System must be cloud-connected to the DataMiner Cloud Platform ([DCP](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html)).
+- The CloudGateway [DxM](https://docs.dataminer.services/user-guide/Cloud_Platform/CloudAdminApp/Managing_cloud-connected_nodes.html) on your Dataminer System must be atleast version 2.9.0.
 
-### Limitations
+### Teams
 
-This chat integration is only supported by cloud-connected DataMiner systems.
+- The Dataminer App must be allowed in your [Teams](https://docs.microsoft.com/en-us/microsoftteams/manage-apps).
 
-### Prerequisites
+## Installation
```

Comment:
I would remove Installation completely and put everything under Examples -> Teams -> Installation
And then below we could have Examples -> Teams -> Usage

Reply:
Examples
├── Teams
│   ├── Installation
│   └── Usage

like this